src/main/java/com/qouteall/immersive_portals/portal/Portal.java

```diff
@@ -165,7 +171,11 @@ public boolean isCullable() {
     public boolean isTeleportable() {
         return teleportable;
     }
-    
+
+    public boolean canInteractThroughPortal() {
```

Comment:
`getInteractable`, and you should add `setInteractable`

A short javadoc (`/**` comment) would be helpful too.

Reply:
Additionally, you actually forgot to respect the config option you just added!

Perhaps add an override under the mirror class?

Reply:
whoops, missed the thing under tick(), but perhaps an override would be more appropriate. Return `interactable` if people have the config option turned on, but if they have it turned off, return false. That way you don't have to mutate the data every tick